test/test_console_script.py

```diff
@@ -0,0 +1,11 @@
+import subprocess
+import unittest
+
+
+class ConsoleScriptTest(unittest.TestCase):
+
+    def test_console_script(self):
+        return_code = subprocess.call("poetry install", shell=True)
+        self.assertEqual(0, return_code, "Poetry install failed")
+        return_code = subprocess.call("poetry run exaslct --help", shell=True)
+        self.assertEqual(0, return_code)
```

Comment:
Would it be possible to also verify std out/std err?

Reply:
Possible yes, but what would be the assertion. The help message gets constructed by a framework from decorators.  Alternative, I could compare the whole help message and change the test every time the help message changes.